VMAccess/vmaccess.py

```diff
@@ -360,9 +367,7 @@ def _set_sshd_config(config, name, val):
             # Match block must be put in the end of sshd config
             break
     if notfound:
-        if i is None:
-            i = 0
-        config.insert(i, "{0} {1}".format(name, val))
```

Comment:
Do we still need this if get_file_contents returns more than one file?

Reply:
The change was because .insert is used for arrays but config is a string so changed it to concat instead